scripts/inventory.gd

```diff
@@ -0,0 +1,44 @@
+extends Control
+onready var itemlist:ItemList = $CanvasLayer/ItemList
+var items:Dictionary = {
+	"Knife": {
+	'picture': 'res://assets/items/knife.png',
+	"description": 'A standard knife',
+	},
+	"Hell Knife": {
+	'picture': 'res://assets/items/hell_knife.png',
+	"description": 'The most powerful knife',
+	},
+	"Wooden Knife": {
+	"description": 'A knife made out of wood, not very effective...',
+	'picture': 'res://assets/items/wooden_knife.png',
+	}
+}
+func _ready():
+	$CanvasLayer.visible = false #<---- I can't do this in editor, so I have to do it via code.
+	additem('Knife')
```

Comment:
I think doing a loop here might be better, than manually adding each item.

Hint: You'll have to first get the list of keys from the dictionary and loop through those.